src/components/molecules/UiDropdown/UiDropdown.stories.js

```diff
@@ -25,6 +25,7 @@ const events = actions({
 export default {
   title: 'Molecules/Dropdown',
   component: UiDropdown,
+  tags: [ '!autodocs' ],
```

Comment:
What's the reason for disabling autodocs in some files?

Reply:
They have a .mdx docs. And in the new version of Storybook it 'autodocs' should be disabled.